unicorn-play/src/test/scala/org/virtuslab/unicorn/StringPlayUnicorn.scala

```diff
@@ -1,8 +1,8 @@
 package org.virtuslab.unicorn
 
 import com.google.inject.{ Inject, Singleton }
-import slick.backend.DatabaseConfig
-import slick.driver.JdbcProfile
+import org.virtuslab.unicorn.SlickExports.JdbcProfile
```

Comment:
As I understand, this change in imports is not backward compatible with previous unicorn 3.1.0 version, am I right?